distributed/deploy/tests/test_adaptive.py

```diff
@@ -347,7 +347,7 @@ def key(ws):
         adaptive = cluster.adapt(minimum=1)
         await adaptive.adapt()
 
-        while len(cluster.scheduler.workers) == 4:
+        while len(cluster.scheduler.workers) >= 3:
```

Comment:
This is unrelated to the tornado changes, but this test was intermittently failing locally.

We're attempting to scale down from four to two workers (based on the grouping key, IIUC), but we might not have torn down both additional workers by the time this loop exits if just checking that we no longer have four workers.